### DISCUSSION TITLE
live Chat helper canned messages / not  showing in chat

### DISCUSSION THREAD

**Discussion by AbouRached** [LiveHelperChat/livehelperchat#1946] [Q&A] [answered]
Hello Guys ,

i'm encountering an issue  with setting up canned messages , i followed the instruction on how to set them up . 
but they aren't showing in the chat interface , even when using an admin account , when pressing # in the chat nothing shows or even when clicking on the canned messages in the chat interface it does nothing 
your support is appreciated . 

![image](https://user-images.githubusercontent.com/115656506/234792741-1a94433a-707e-4d40-945f-cecdadd40dab.png)
![image](https://user-images.githubusercontent.com/115656506/234792918-ad779684-359f-4a6a-8f22-57bd60bc5e17.png)
![image](https://user-images.githubusercontent.com/115656506/234793065-7b9389c9-5c69-451f-8e00-cb9671aa9359.png)
![image](https://user-images.githubusercontent.com/115656506/234793153-aa42af57-069b-49ff-b193-2eea267fb60d.png)

**Comment by remdex** [comment]
Please check `cache/default.log` most likely your mysql server does not support `json_`. or you just have misconfigured :)

**Reply by AbouRached** [reply]
Thank you , yes i checked seems that is the issue , i am running mysql  Ver 15.1 Distrib 5.5.68-MariaDB, for Linux (x86_64) 
, any tips of what mysql server is the most efficient to use ?

**Reply by remdex** [reply]
It does not matter as long you have `json_` functions support in MySQL...

**Reply by AbouRached** [reply, marked as answer]
this function is available since v-5.7 of mysql, I had a lower version of mysql.

https://stackoverflow.com/questions/40150761/mysql-error-code-1305-function-json-extract-does-not-exist
